css/stylesheet.css

```diff
@@ -0,0 +1,803 @@
+* {
+  -webkit-box-sizing: border-box;
+     -moz-box-sizing: border-box;
+          box-sizing: border-box;
+}
+@font-face {
+  font-family: 'icomoon';
+  src:url('fonts/icomoon.eot?-lo05z1');
+  src:url('fonts/icomoon.eot?#iefix-lo05z1') format('embedded-opentype'),
+    url('fonts/icomoon.ttf?-lo05z1') format('truetype'),
+    url('fonts/icomoon.woff?-lo05z1') format('woff'),
+    url('fonts/icomoon.svg?-lo05z1#icomoon') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+[class^="icon-"], [class*=" icon-"] {
+  font-family: 'icomoon';
+  speak: none;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 3;
+  font-size: 24px;
+ -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+.p-services [class^="icon-"], [class*=" icon-"] {
+  background-color: white;
+  color: black;
+  margin: auto;
+}
+.icon-paint-format:before {
+  content: "\e605";
+}
+.icon-cog:before {
+  content: "\e606";
+}
+.icon-lab:before {
+  content: "\e607";
+}
+.icon-google-plus:before {
+  content: "\e600";
+  background-color: #dd4b39;
+}
+.icon-facebook:before {
+  content: "\e601";
+  background-color: #3b5998;
+}
+.icon-twitter:before {
+  content: "\e602";
+  background-color: #00aced;
+}
+.icon-dribbble:before {
+  content: "\e603";
+  background-color: #ea4c89;
+}
+.icon-linkedin2:before {
+  content: "\e604";
+  background-color: #007bb6;
+}
+.icons {
+  margin: auto;
+  margin-bottom: 10%;
+  width: 100%;
+  text-align: center;
+}
+.icons span{
+  color: white;
+  display: inline-block;
+  -webkit-transition: -webkit-transform .25s ease-in-out;
+  transition: transform .25s ease-in-out;
+  line-height: 1;
+}
+.icons span:hover {
+  border: white;
+  -ms-transform: scale(1.25,1.25);
+  -webkit-transform: scale(1.25,1.25);
```

Comment:
The spec is to only support recent versions of browsers. There is no need for the `ms` vendor prefixes. 

http://caniuse.com/#search=transform